Formula/biointerchange.rb

```diff
@@ -0,0 +1,35 @@
+class Biointerchange < Formula
+  desc "Genomic linked data converter: an interchange GFF, GVF, VCF files via JSON"
+  homepage "https://github.com/BioInterchange/BioInterchangeC"
+  url "http://indie.kim/assets/biointerchange/biointerchange-2.0.5.tar.gz"
+  sha256 "6837b4564ebe996d552de552afe40ac28314a3876c3161c8735f3956c494bb57"
+  license "MIT"
+
+  depends_on "autoconf" => :build
+  depends_on "cmake" => :build
+  depends_on "googletest" => :test
+  depends_on "curl"
+  depends_on "gettext"
+  depends_on "libiconv"
+  depends_on "openssl@1.1"
+  depends_on "python@3.9"
+
+  def install
+    # NOTE: Cannot parallelize, because:
+    # 1. OpenSSL, curl, and LibDocument are all ExternalProjects
+    # 2. LibDocument also pulls & builds CPython
+    ENV.deparallelize
+    system "cmake", "-DCMAKE_BUILD_TYPE=Release", "-G", "Unix Makefiles"
+    system "make", "libdocument-lib"
+    system "make", "biointerchange"
+    system "strip", "biointerchange"
```

Comment:
This command should not be necessary. The result of `make biointerchange` should be built without debugging symbols thanks to Superenv, which removes `-g` from `gcc -g`.

Reply:
We typically do not run strip for Homebrew. I'd prefer to omit it.

Reply:
The default seems to be that Homebrew's build doesn't strip. I have removed the line from the Formula, but it's strange from a C/C++ dev perspective. It's actively leaving junk in a release product.

Reply:
Homebrew's Superenv removes debugging symbols by removing `-g` from `gcc -g`.

Reply:
You aren't wrong, but the additional symbols come from external sources and are added when linking.